DependencyInjection/CmfRoutingExtension.php

```diff
@@ -82,7 +87,20 @@ private function setupDynamicRouter(array $config, ContainerBuilder $container,
         $loader->load('routing-dynamic.xml');
 
         // strip whitespace (XML support)
-        foreach (array('controllers_by_type', 'controllers_by_class', 'templates_by_class', 'route_filters_by_id') as $option) {
+        foreach (array('controllers_by_type', 'controllers_by_class', 'templates_by_class') as $option) {
+            $config[$option] = array_map(function ($items) {
+                foreach ($items as $key => $item) {
+                    if (is_array($item)) {
+                        $items[$key]['value'] = trim($item['value']);
+                    } else {
+                        $items[$key] = trim($item);
```

Comment:
by the normalizer, right?

Am 21.05.2016 um 09:54 schrieb David Buchmann:
>
> In DependencyInjection/CmfRoutingExtension.php
> <https://github.com/symfony-cmf/routing-bundle/pull/349#discussion_r64130892>:
>
> > @@ -82,7 +87,20 @@ private function setupDynamicRouter(array $config, ContainerBuilder $container,
> >          $loader->load('routing-dynamic.xml');
> >  
> >          // strip whitespace (XML support)
> > -        foreach (array('controllers_by_type', 'controllers_by_class', 'templates_by_class', 'route_filters_by_id') as $option) {
> > +        foreach (array('controllers_by_type', 'controllers_by_class', 'templates_by_class') as $option) {
> > +            $config[$option] = array_map(function ($items) {
> > +                foreach ($items as $key => $item) {
> > +                    if (is_array($item)) {
> > +                        $items[$key]['value'] = trim($item['value']);
> > +                    } else {
> > +                        $items[$key] = trim($item);
>
> its indeed the xml configuration. we have tests making sure that
> controllers with whitespace around them do work. surprised though that
> this is not in the Configuration class instead.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/symfony-cmf/routing-bundle/pull/349/files/5964ebe6047f649f9defc57b686e0bc169b17213#r64130892>
>